gemelli/rpca.py

```diff
@@ -39,6 +42,11 @@ def phylogenetic_rpca(table: biom.Table,
        gemelli.
     """
 
+    # loads NewickFormat tree as bp.BP tree; loads & parses tree faster
+    phylogeny = get_bp(phylogeny)
```

Comment:
`get_bp` is only like two lines and it is pretty easy to get burned by importing from private modules in python. I would probably recommend just duplicating that code here.